README.md

```diff
@@ -138,6 +139,7 @@ Official releases of DC/OS can be found at <http://dcos.io/releases/>.
 By default, DC/OS Docker (`./configure`) downloads the latest **stable** version of DC/OS.
 
 To use a different version, specify the path to the installer interactively when running `./configure` or manually in `make-config.mk` by setting `DCOS_GENERATE_CONFIG_PATH`.
+The installer file must be located in (or under) the `dcos-docker` directory.
```

Comment:
This is not the case with `Make`. `./configure` needs a relative path, but `DCOS_GENERATE_CONFIG_PATH` can be set to anywhere on the filesystem.

I ran the following and it used the installer just fine:

```
> make DOCKER_VERSION=1.13.1 MESOS_SYSTEMD_ENABLE_SUPPORT=false DCOS_GENERATE_CONFIG_PATH=/tmp/dcos_generate_config.ee.sh 
```

Either change this to say that `When using ./configure...` or remove this change.

Reply:
There's some vagaries around this when using dcos-docker in a VM, but otherwise you're correct. You can set `DCOS_GENERATE_CONFIG_PATH` to point to anywhere, as long as you can read it from the local machine as the current user.

That said, `./configure` doesn't even pay attention to `DCOS_GENERATE_CONFIG_PATH` as an environment variable and there's no way to pass a make var to the bash script. What it actually does is prompts you for a relative path saying `Enter DC/OS installer path` and then uses that to set `DCOS_GENERATE_CONFIG_PATH` by prefixing it with `$(CURDIR)/`. If you really want to make that more obvious, just go update the prompt to say `Enter DC/OS installer path (relative to $(CURDIR))` or something similar.

Reply:
To summarise:
- If using Vagrant/VirtualBox, the installer must be under the `dcos-docker` directory.
- If using the workflow documented in the README, the installer must be under the `dcos-docker` directory.
- Once `configure` has been run with an installer under the `dcos-docker` directory, it is possible to modify the generated `make_config.mk` file to point to an installer elsewhere, or use `make DCOS_GENERATE_CONFIG_PATH=...`.

I'll add a rephrasing of this last line to the README.

Reply:
I'm not familiar with the `./configure` workflow, and I'll let @karlkfi speak to the intentions of that. However, my understanding is that the above is not quite right. I think that there are at least two interfaces that we want to work:

1. Using `./configure`
2. Using `make` directly, as I did.

Note that in the example I posted above, there is no use of `./configure`.

> If using Vagrant/VirtualBox, the installer must be under the dcos-docker directory.

Really? I don't use this setup but I don't think that this is the case. In fact, I think that @karlkfi 's comment confirms that it is not - "There's some vagaries around this when using dcos-docker in a VM, but otherwise you're correct. You can set DCOS_GENERATE_CONFIG_PATH to point to anywhere, as long as you can read it from the local machine as the current user.".

I think that two options are:
* Do not modify anything
* Modify the prompt in `./configure` as suggested by @karlkfi

Reply:
Please check the latest version, to see if that accords with your understanding. If not, I will remove the change altogether. The intent of the change was to clarify, so if it is not doing that, it's not worth doing.

Reply:
LGTM